recipes/bioconductor-marray/meta.yaml

```diff
@@ -3,10 +3,10 @@ package:
   version: 1.50.0
 source:
   fn: marray_1.50.0.tar.gz
-  url: http://bioconductor.org/packages/release/bioc/src/contrib/marray_1.50.0.tar.gz
-  md5: 32c11e22e20003bc1c422832f5dae7bc
+  url: https://bioarchive.galaxyproject.org/marray_1.50.0.tar.gz
+  md5: c068e960357aa8d9d84b9f517dd1ef50
 build:
-  number: 0
+  number: 1
```

Comment:
bioconductor update script bumps build number if the version is the same but URL and md5sum change. Bumping build number ensures the recipe gets built with the new URL. I'm fine either way, but feel like the bump keeps the recipe and package in sync.